docs/en/toolkit/frameworks.md

```diff
@@ -7,10 +7,12 @@ lang: en
 
 # Toolkit Frameworks
 
-Toolkit frameworks are collections of code that can be used across apps and engines. The Shotgun Utilities and QT Widgets frameworks are especially useful in app development.
```

Comment:
I think I added the "The Shotgun Utilities and QT Widgets frameworks are especially useful in app development." sentence, and my reasoning was that this is not an exhaustive list of *all* FWs that we provide, and *not* mentioning anything about that can be deceiving. I wanted to make mention of the fact that we're calling out two specific FWs here. I'll leave it up to you, but I'd consider adding this sentence back in after the "We also provide some... " sentence below.

Reply:
Thats fair, but do we provide public documentation for the other frameworks? Are they intended to be used by the public?

Reply:
How about "We provide a public API for the following frameworks which can be used in app development:"

Reply:
I dig it! Be sure to add a comma before "which".